live-examples/js-examples/number-min-value.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">function multiply(x,y) {
+  if (x * y < Number.MIN_VALUE) { return ("Process as -Infinity")};
```

Comment:
Any specific reason the string is wrapped in parentheses?

Reply:
Nope - just a mistake :) We should also maybe think about where we use double quotes as opposed to single. For example airbnb's JS code style is to have them in single quotes: https://github.com/airbnb/javascript#strings

Reply:
Agree, I definitely vote for single quotes over double.